docs/endpoints/scan-results.md

```diff
@@ -24,35 +24,63 @@ The body must contain all collected data from the client's device.
   },
   "beaconLogs": [
     {
-      "major": 16,
       "UUID": "44c0ffee-988a-49dc-0bad-a55c0de2d1e4",
+      "major": 16,
+      "minor": 0,
       "timestamp": 1406804464.128,
+      "timezone": "UTC+01:00",
       "rssi": -60,
-      "id": 1,
-      "minor": 0,
       "beacon_id": 16
     }, {
-      "major": 16,
       "UUID": "44c0ffee-988a-49dc-0bad-a55c0de2d1e4",
+      "major": 16,
+      "minor": 0,
       "timestamp": 1406804474.128,
+      "timezone": "UTC+02:00",
       "rssi": -60,
-      "id": 2,
-      "minor": 0,
       "beacon_id": 16
     }
   ]
 }
 ```
 
+| Parameter               | Description                                              | Required |
```

Comment:
@adri not sure where you got
```
The UTC timezone format can be also expressed like UTC-01 (only hours)
```
but that is an invalid UTC timezone (offset) format.